### PR TITLE
enh/libs-utils

### PR DESCRIPTION
**Pull request by michael-scheurer**

### CODE REVIEW

**Review by Itrulia — APPROVED** [review]

**Review by Itrulia — APPROVED** [review]

File: apps/editor/__tests__/specs/__snapshots__/utility.test.ts.snap [hunk 1]
```diff
@@ -54,7 +54,7 @@ exports[`slugify should transform text with special r like ŔŖŘ correctly 1`]
 
 exports[`slugify should transform text with special s like ŚŜŞŠ correctly 1`] = `"text-with-special-s-like-ssss"`;
 
-exports[`slugify should transform text with special ss like ß correctly 1`] = `"text-with-special-ss-like-ss"`;
+exports[`slugify should transform text with special ss like ß correctly 1`] = `"text-with-special-ss-like-s"`;
```
Comment (Itrulia) [inline apps/editor/__tests__/specs/__snapshots__/utility.test.ts.snap:57]: should this be ss or s? ß should be ss technically. One slugify method did this correctly, the other didn't.

**Review by Itrulia — CHANGES_REQUESTED** [review]
ß question open, else approved

**Review by Itrulia — APPROVED** [review]

File: apps/editor/__tests__/specs/__snapshots__/utility.test.ts.snap [hunk 2]
```diff
@@ -68,4 +68,4 @@ exports[`slugify should transform text with special y like ÝŶŸỲỴỶỸ co
 
 exports[`slugify should transform text with special z like ŹŻŽ correctly 1`] = `"text-with-special-z-like-zzz"`;
 
-exports[`slugify should transform text with umlaute like ä ö and ü correctly 1`] = `"text-with-umlaute-like-ae-oe-and-ue"`;
+exports[`slugify should transform text with umlaute like ä ö and ü correctly 1`] = `"text-with-umlaute-like-a-o-and-u"`;
```
Comment (Itrulia) [inline apps/editor/__tests__/specs/__snapshots__/utility.test.ts.snap:71]: Sorry I missed it earlier, same thing here: https://wepublish.atlassian.net/browse/WPC-242

**Review by Itrulia — CHANGES_REQUESTED** [review]

**Comment by tomaszdurka** [comment]
@michael-scheurer can you update this one quickly, get review and merge please ? :)

**Comment by michael-scheurer** [comment]
> @michael-scheurer can you update this one quickly, get review and merge please ? :)

@tomaszdurka I'm not able to merge, since the tests are failing on master. Could you fix that first? Every thing else is done in that branch and is now ready to be merged.

**Comment by tomaszdurka** [comment]
@michael-scheurer OK. Thanks for merging. Will wait for master to be fixed.

**Review by LukasSkywalker — DISMISSED** [review]

**Review by Itrulia — APPROVED** [review]

**Pull request closed without merging**